packages/uikit/src/components/CurrencyLogo/utils.ts

```diff
@@ -10,9 +10,9 @@ const mapping: { [key: number]: string } = {
 export const getTokenLogoURL = memoize(
   (token?: Token) => {
     if (token && mapping[token.chainId]) {
-      return `https://assets-cdn.trustwallet.com/blockchains/${mapping[token.chainId]}/assets/${getAddress(
+      return `/images${token.chainId === ChainId.BSC ? "" : `/${token.chainId}`}/tokens/${getAddress(
```

Comment:
We only have farm logo hosted on our side. The currencylogo is general for all possible tokens

Reply:
ah ok then, I'll close this one